tests/mentorship_relation/test_api_accept_request.py

```diff
@@ -39,13 +40,59 @@ def setUp(self):
     def test_accept_mentorship_request(self):
         self.assertEqual(MentorshipRelationState.PENDING, self.mentorship_relation.state)
         with self.client:
-            response = self.client.put('/mentorship_relation/%s/accept' % self.mentorship_relation.id,
+            response = self.client.put(f"/mentorship_relation/{self.mentorship_relation.id}/accept",
                                        headers=get_test_request_header(self.second_user.id))
 
             self.assertEqual(200, response.status_code)
             self.assertEqual(MentorshipRelationState.ACCEPTED, self.mentorship_relation.state)
             self.assertDictEqual(messages.MENTORSHIP_RELATION_WAS_ACCEPTED_SUCCESSFULLY,
-                             json.loads(response.data))
+                                 json.loads(response.data))
+
+    def test_accept_mentorship_request_user_already_in_relation(self):
+        self.assertEqual(MentorshipRelationState.PENDING, self.mentorship_relation.state)
+        with self.client:
+            response = self.client.put(f"/mentorship_relation/{self.mentorship_relation.id}/accept",
+                                       headers=get_test_request_header(self.second_user.id))
+
+            self.assertEqual(200, response.status_code)
+            self.assertEqual(MentorshipRelationState.ACCEPTED, self.mentorship_relation.state)
+            self.assertDictEqual(messages.MENTORSHIP_RELATION_WAS_ACCEPTED_SUCCESSFULLY,
+                                 json.loads(response.data))
+
+            # Main part of the test - user 2 tries to accept relation request again
+            response = self.client.put(f"/mentorship_relation/{self.mentorship_relation.id}/accept",
+                                       headers=get_test_request_header(self.second_user.id))
+
+            self.assertEqual(400, response.status_code)
+            self.assertEqual(MentorshipRelationState.ACCEPTED, self.mentorship_relation.state)
+            self.assertDictEqual(messages.NOT_PENDING_STATE_RELATION,
```

Comment:
This test should return USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION

Reply:
So should I alter API code in `/app`? Because the only place where `USER_IS_INVOLVED_IN_A_MENTORSHIP_RELATION` is `TaskDAO` in `app/api/dao/task.py`.

Reply:
It is present in `app/api/dao/mentorship_relation.py`.Not sure if this is what you are asking?

Reply:
Okay, my bad, I must have missed something

Reply:
Fixed this one:)